go.mod

```diff
@@ -105,6 +106,7 @@ require (
 )
 
 replace (
+	github.com/emicklei/go-restful => github.com/emicklei/go-restful/v3 v3.8.0
```

Comment:
what's the reason behind pinning this package?

Reply:
To fix the CVE-2022-1996, we must use go-restful v3.8 or higher, but the metrics.go 0.24 import github.com/emicklei/go-restful, so we replace it to v3